pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -61,15 +61,15 @@ type CronWorkflowSpec struct {
 	Timezone string `json:"timezone,omitempty" protobuf:"bytes,8,opt,name=timezone"`
 	// WorkflowMetadata contains some metadata of the workflow to be run
 	WorkflowMetadata *metav1.ObjectMeta `json:"workflowMetadata,omitempty" protobuf:"bytes,9,opt,name=workflowMeta"`
-	// StopStrategy defines if the cron workflow will stop being triggered once a certain condition has been reached, involving a number of runs of the workflow
+	// v3.6 and after: StopStrategy defines if the CronWorkflow should stop scheduling based on a condition
 	StopStrategy *StopStrategy `json:"stopStrategy,omitempty" protobuf:"bytes,10,opt,name=stopStrategy"`
 	// Schedules is a list of schedules to run the Workflow in Cron format
 	Schedules []string `json:"schedules,omitempty" protobuf:"bytes,11,opt,name=schedules"`
 }
 
-// StopStrategy defines if the cron workflow will stop being triggered once a certain condition has been reached, involving a number of runs of the workflow
+// v3.6 and after: StopStrategy defines if the CronWorkflow should stop scheduling based on a condition
 type StopStrategy struct {
-	// Condition defines a condition that stops scheduling workflows when evaluates to true. Use the
+	// v3.6 and after: Condition is an expression that stops scheduling workflows when true. Use the
 	// keywords `failed` or `succeeded` to access the number of failed or successful child workflows.
```

Comment:
```suggestion
	// variables `failed` or `succeeded` to access the number of failed or successful child workflows.
```
same as above, "variables" is the existing term used in the docs